qos_capacity.adoc

```diff
@@ -440,36 +439,36 @@ blocks, respectively.
 
 <<<
 
-The `FLUSH_RCID` operation may incur a long latency to complete. New requests to
-the controller by the `RCID` being flushed are allowed. Additionally, the
+The `FLUSH_RCID` operation can incur a long latency to complete. New requests to
+the controller by flushing the `RCID` are allowed. Additionally, the
```

Comment:
@kersten1 The original sentence was stating that while the process of flushing the RCID is in progress, new requests to the by the RCID which is in the process of being flushed are allowed.